app/vmctl/flags.go

```diff
@@ -10,8 +10,9 @@ import (
 )
 
 const (
-	globalSilent  = "s"
-	globalVerbose = "verbose"
+	globalSilent             = "s"
+	globalVerbose            = "verbose"
+	globalDisableProgressBar = "disable-progress-bar"
```

Comment:
I think this flag needs to be passed to `barpool` subpackage. When set, all operations with bar needs to be no-op. This would simplify bar usage for all other packages.

Reply:
Use this flag as a param when initializing progress bar